src/components/movie/SingleMoviePage.vue

```diff
@@ -140,23 +141,24 @@ export default {
       return Math.floor(this.movie.vote_average);
     },
   },
-  async created() {
-    try {
-      this.isLoading = true;
-      this.getActors();
-      this.getImgs();
-      this.getReviews();
-      this.isLoading = false;
-    } catch (e) {
-      this.isLoading = false;
-      console.log(e);
-    }
+  created() {
+    this.getData();
   },
   methods: {
     ...mapActions(["getMovieImages", "getMovieActors", "getMovieReviews", "changeMovieReviewsPage"]),
     toMovieHomepage(url) {
       window.location.href = url;
     },
+    async getData() {
+      try {
+        this.isLoading = true;
+        await Promise.all([this.getImgs(), this.getActors(), this.getReviews()]);
```

Comment:
Better
```javascript
await this.getImgs();
await this.getActors();
await this.getReviews();
```
Read this [topic](https://learn.javascript.ru/promise-api#:~:text=%D0%9C%D0%B5%D1%82%D0%BE%D0%B4%20Promise.,%D1%80%D0%B5%D0%B7%D1%83%D0%BB%D1%8C%D1%82%D0%B0%D1%82%D0%BE%D0%BC%20%D0%B1%D1%83%D0%B4%D0%B5%D1%82%20%D0%BC%D0%B0%D1%81%D1%81%D0%B8%D0%B2%20%D0%B8%D1%85%20%D1%80%D0%B5%D0%B7%D1%83%D0%BB%D1%8C%D1%82%D0%B0%D1%82%D0%BE%D0%B2.) why